static/app/components/u2f/u2finterface.tsx

```diff
@@ -187,7 +165,11 @@ class U2fInterface extends React.Component<Props, State> {
     let promise: Promise<u2f.SignResponse | u2f.RegisterResponse>;
     if (this.props.flowMode === 'sign') {
       if (this.props.isWebauthnSigninFFEnabled) {
-        this.webAuthnSignIn(this.props.challengeData.authenticateRequests);
+        const challengeArray = base64urlToBuffer(
+          this.props.challengeData.webAuthnAuthenticationData
+        );
+        const challenge = cbor.decodeAllSync(challengeArray);
+        this.webAuthnSignIn(challenge[0]);
```

Comment:
probably worth adding another comment about only using the first challenge

Reply:
i added it in https://github.com/getsentry/sentry/pull/30351 if thats alright